mula/scheduler/storage/stores/pq.py

```diff
@@ -85,6 +85,29 @@ def pop_boefje(
 
             return items
 
+    def get_possible_duplicate_boefje_tasks(
+        self, scheduler_id: str, boefje_id: str, input_ooi: str
+    ) -> list[models.Task] | None:
```

Comment:
Just wondering; is there a specific reason to distinguish between a `None` and empty list as a result type? After all, if a database query doesn't yield anything that's still a valid result imo